src/main/java/emissary/admin/Startup.java

```diff
@@ -413,6 +423,21 @@ protected void stopAndWaitForPlaceCreation() {
             numPlacesFound = this.places.size();
 
             if (numPlacesFound >= numPlacesExpected) {
+
+                if (Boolean.parseBoolean(System.getProperty("strict.mode")) &&
+                        this.failedPlaces.size() >= 1) {
+
+                    StringBuilder failedPlaceList = new StringBuilder();
+                    failedPlaceList.append("The following places have failed to start: \n");
+                    for (String s : this.failedPlaces) {
+                        failedPlaceList.append(s + "\n");
+                    }
+                    logger.error(failedPlaceList.toString());
+                    logger.error(("Server shutting down"));
+                    System.exit(1);
```

Comment:
- [ ] Need to get clarification on the preferred way to abort the startup.  This is probably sufficient, but I'd like some additional confirmation. (We may want to also write something to `System.out`/`System.err`)

Reply:
I chose this method since a non-zero exit code indicates an abnormal exit in java, and that's the code we used in a few other instances of the system erroring out.  But yeah, I'd like at least one other person to confirm this is the way it should be done.

Reply:
> I don’t have an answer. But something to consider is when this is running in K8s I would prefer to see the pod have an error and not be in a restart loop.
> 
> It would be a real pain if this kind of error did not prevent a rollout restart from continuing to all replicas.

That's a strong argument for being able to run a canary deployment of some kind.  _(Obviously, that's far outside the scope of this PR.)_